mistral/mixtral-8x7b-instruct-vllm/model/model.py

```diff
@@ -0,0 +1,35 @@
+import subprocess
+import uuid
+
+from vllm import SamplingParams
+from vllm.engine.arg_utils import AsyncEngineArgs
+from vllm.engine.async_llm_engine import AsyncLLMEngine
+
+
+class Model:
+    def __init__(self, **kwargs):
+        self.model = None
+        self.llm_engine = None
+
+        # Install megablocks and run ray process
+        subprocess.run(["pip", "install", "megablocks"])
```

Comment:
why install this here instead of requirements?

Reply:
`megablocks` needs a GPU build machine. This is a temp workaround.